NEWS.md

```diff
@@ -1,6 +1,16 @@
+
 # hint 1.14.0
 * Hide project history column(shared by) when no data
```

Comment:
This comment should be updated to more clearly reflect the actual changes now being made.